mlflow/java/client/src/main/java/org/mlflow/client/objects/ToProtobufMapper.java

```diff
@@ -0,0 +1,71 @@
+package org.mlflow.client.objects;
+
+import java.util.*;
+
+import com.google.protobuf.MessageOrBuilder;
+import com.google.protobuf.util.JsonFormat;
+
+import org.mlflow.api.proto.Service.*;
+
+public class ToProtobufMapper {
```

Comment:
Does this need to be public or just package private?

Reply:
It needed to be public because I think Java doesn't have `private[mlflow]` syntax. I moved it to super package and made package-private though.